File: src/main/java/com/alan/cockapooserver/blog/Blog.java
package com.alan.cockapooserver.blog;


import jakarta.persistence.*;
import lombok.Data;
import org.hibernate.annotations.ColumnDefault;

@Entity
@Data
public class Blog {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column()
    private String name;
    @Column()
    private String url;
    @Column()
    @ColumnDefault("0")
    private Integer views;

}

File: src/main/kotlin/com/example/cockapoo/cockapoo_server/domain/entity/Blog.kt


import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import org.springframework.data.annotation.Id
import java.time.LocalDateTime

@Entity
class Blog(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var title: String,
    var content: String?,
    var author: String?,
    var link: String?,
    var viewCount: Long = 0,
    var createdAt: LocalDateTime = LocalDateTime.now(),
    var updatedAt: LocalDateTime = LocalDateTime.now()
)